src/frontend/admin/src/utils/forms.ts

```diff
@@ -29,7 +29,10 @@ export const appendToFormData = (
 };
 
 export const exportToFormData = (payload: any): FormData | any => {
-  if (process.env.NEXT_PUBLIC_API_SOURCE === "mocked") {
+  if (
```

Comment:
a more concise syntax ? 

```suggestion
if (["mocked", "test"].includes(process.env.NEXT_PUBLIC_API_SOURCE))
```